classroom/snippets/src/main/java/CreateAlias.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// [START classroom_create_alias]
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.services.classroom.Classroom;
+import com.google.api.services.classroom.ClassroomScopes;
+import com.google.api.services.classroom.model.Course;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import java.io.IOException;
+import java.util.Collections;
+
+/* Class to demonstrate how to create a course with an alias. */
+public class CreateAlias {
+  /**
+   * Create a new course with an alias. Set the new course id to the desired alias.
+   *
+   * @return - newly created course.
+   * @throws IOException - if credentials file not found.
+   */
+  public static Course createAlias() throws IOException {
```

Comment:
This method looks like it's creating a Course with an Alias already attached, but not creating an Alias; is my understanding correct?

Reply:
Got it. I think we might want to rename some of these then, perhaps "createCourseWithAlias" would be a better name here. It's better to be verbose and clear in these examples as a hint to readers as to the behavior they expect. Similarly maybe AddAlias should be something like "AddAliasToCourse".